### PR TITLE
Dedup test fix

### PR DESCRIPTION
**Pull request by erezh31**
Fix the playbook test of our dedup scripts

### CODE REVIEW

**Comment by erezh31** [comment]
Note that I did not make any changes in the script, so the CHANGELOG is empty

**Comment by Itay4** [comment]
@erezh31 please mention which files should @kirbles19 review (playbooks that require descriptions text review, changelogs..)

**Comment by erezh31** [comment]
@kirbles19 plz review descriptions, release notes, thx!

**Comment by erezh31** [comment]
@Itay4 I made changes to playbook-Dedup_-_Generic.yml, and playbook-Dedup_-_Generic_-_Test.yml  which is the playbook tet

File: Playbooks/playbook-Dedup_-_Generic.yml [hunk 1]
```diff
@@ -346,6 +370,12 @@ inputs:
     simple: "true"
   required: false
   description: If duplicate found - close the incident as duplicate (true\false)
+- key: Time field
```
Comment (Itay4) [inline Playbooks/playbook-Dedup_-_Generic.yml]: i think its best practise not to have white spaces in inputs, should be `TimeField`

**Comment by kirbles19** [comment]
@erezh31 All done.

**Comment by erezh31** [comment]
@Itay4 CR fixed, plz approve

**Pull request merged**